### DISCUSSION TITLE
Help for testing!

### DISCUSSION THREAD

**Discussion by tetenc555** [tetenc555/Opencore-ThinkPad-T480#5] [General]
Please use this to post about your testing with this EFI! Feedback is always greatly appreciated, principally if you tried out the things on the Not Tested section!

**Comment by galaxy-nomad** [comment]
Has the built in ethernet port ever worked for you on any version of macos?

**Reply by Lazlow7723** [reply]
Yes. I'm on Sequoia 15.7.4 and Ethernet works fine.

**Reply by galaxy-nomad** [reply]
Ok thank you for confirming. I think my ethernet port might be dead as I get no internet connection when plugging in directly
